website/home/app/showcase/showcase-client.tsx

```diff
@@ -0,0 +1,389 @@
+"use client";
+
+import { motion } from "framer-motion";
+import Image from "next/image";
+import Link from "next/link";
+import AnimatedTitle from "@/components/ui/AnimatedTitle";
+import { ExternalLink } from "lucide-react";
+
+// Showcase data structure - easy to add new items
+interface ShowcaseItem {
+  id: string;
+  title: string;
+  description: string;
+  image?: string;
+  demo?: string; // Link to live demo
+  github?: string; // Link to GitHub repo
+  tags: string[];
+}
+
+const showcaseItems: ShowcaseItem[] = [
+  {
+    id: "chat-ux",
+    title: "Chat UX Template",
+    description:
+      "Next.js chat interface with real-time streaming responses using GenSX workflows and server-sent events.",
+    demo: "https://gensx-chat-ux.vercel.app",
```

Comment:
i think we're misisng the `image` property here

Reply:
We should also setup a proper domain for this, like chat-demo.gensx.dev/com or something

Reply:
> i think we're misisng the image property here
Updated 

> We should also setup a proper domain for this, like chat-demo.gensx.dev/com or something
Agreed but I'll get this pushed and we can do that as a separate pr to follow up